### PR TITLE
Fix Namespace Schema Compilation Error

### PR DESCRIPTION
**Pull request by bbenetskyy**
Same as Described in [This Article](https://andreinitescu.blogspot.com/2019/02/xamarinforms-xmlnsdefinition-attribute.html) I was not able to compile and run the solution at the latest `master` branch.

These two lines - fix that problem for me

image

### CODE REVIEW

**Comment by mgierlasinski** [comment]
What if you reference MagicGradients from NuGet?

**Comment by mgierlasinski** [comment]
We are missing Init() in library to prevent linker from stripping types:
https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/custom-namespace-schemas#consuming-a-custom-namespace-schema

**Comment by mgierlasinski** [comment]
Please use tha same solution as described here:
https://github.com/xamarin/XamarinCommunityToolkit/pull/357

File: MagicGradients.Toolkit/Build/MagicGradientsToolkit.targets [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<Project>
+    <ItemGroup>
+        <Compile Include="$(MSBuildThisFileDirectory)PreserveMagicGradientsToolkitAttribute.cs"
```
Comment (mgierlasinski) [inline MagicGradients.Toolkit/Build/MagicGradientsToolkit.targets:3]: Please name it PreserveMagicToolkitAttribute and everything related for shorter name :)

File: MagicGradients.Toolkit/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -2,5 +2,5 @@
 using Xamarin.Forms.Internals;
 
 [assembly: Preserve]
-[assembly: XmlnsDefinition("http://magic.com/schemas/toolkit", "MagicGradients.Toolkit")]
-[assembly: XmlnsDefinition("http://magic.com/schemas/toolkit", "MagicGradients.Toolkit.Controls")]
+[assembly: XmlnsDefinition("http://magic.com/schemas/2021/toolkit", "MagicGradients.Toolkit")]
+[assembly: XmlnsDefinition("http://magic.com/schemas/2021/toolkit", "MagicGradients.Toolkit.Controls")]
```
Comment (mgierlasinski) [inline MagicGradients.Toolkit/Properties/AssemblyInfo.cs:6]: We agreed to have http://magic.com/schemas/toolkit, please revert

File: MagicGradients/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -5,6 +5,6 @@
 [assembly: InternalsVisibleTo("MagicGradients.Tests")]
 
 [assembly: Preserve]
-[assembly: XmlnsDefinition("http://magic.com/schemas/gradients", "MagicGradients")]
-[assembly: XmlnsDefinition("http://magic.com/schemas/gradients", "MagicGradients.Animation")]
-[assembly: XmlnsDefinition("http://magic.com/schemas/gradients", "MagicGradients.Masks")]
+[assembly: XmlnsDefinition("http://magic.com/schemas/2021/gradients", "MagicGradients")]
+[assembly: XmlnsDefinition("http://magic.com/schemas/2021/gradients", "MagicGradients.Animation")]
+[assembly: XmlnsDefinition("http://magic.com/schemas/2021/gradients", "MagicGradients.Masks")]
```
Comment (mgierlasinski) [inline MagicGradients/Properties/AssemblyInfo.cs:10]: We agreed to have http://magic.com/schemas/gradients, please revert

**Comment by bbenetskyy** [comment]
## Just Build or Clean/ReBuild

Rider Compilation Errors:
image

Visual Studio Compilation Errors:
image

## Try to Run Playground on iOS

The Rider has only warnings. The application runs without any problems.
Visual Studio has only warnings. The application runs without any problems.

## Try to Run Playground on Android

Rider - Same Compilation Error. The application not deployed to Emulator.

Visual Studio has only warnings. The application runs without any problems.






# Conclusion - no more changes are needed, we can merge it

**Pull request merged**